build.sbt

```diff
@@ -1,5 +1,5 @@
 /*
```

Comment:
I still don't see changes for [CI/CD ticket](https://github.com/snowplow/sauna/issues/98). Let me know if you need any help with it. And make sure all other issues of milestone are present.